installer/Migration/V5_2_0/lang-string/buzz.yaml

```diff
@@ -0,0 +1,7 @@
+langStrings:
+  -
+    value: 'Upcoming Anniversaries'
+    unitId: upcoming_anniversaries
```

Comment:
check with BA do we need to add source notes. Related: OHRM5X-1644, #1463

Reply:
Said no need to add